scrapy_zyte_api/_params.py

```diff
@@ -19,6 +19,7 @@
     "article",
     "articleList",
     "articleNavigation",
+    "jobPosting",
```

Comment:
Noticed it lacking while working on this, and I thought I would added given how trivial it is.

I hope we can come up with a solution to minimize the chance of forgetting things like this as part of https://github.com/scrapy-plugins/scrapy-zyte-api/issues/143.